commit ec79905fb214a3430f213b9e1c2725c697781477 (refs/changes/24/1724/1)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-03-27T13:52:24+02:00 (2 years, 6 months ago)
    
    TOOLS-1730 Add adminui to the list of HA-eligible services

diff --git a/lib/procedures/index.js b/lib/procedures/index.js
index e07cadd..f81f7d2 100644
--- a/lib/procedures/index.js
+++ b/lib/procedures/index.js
@@ -27,6 +27,7 @@ var errors = require('../errors'),
 
 // --- HA ready services
 var HA_READY_SVCS = [
+    'adminui',
     'cloudapi',
     'cmon',
     'mahi',
